jdaviz/configs/mosviz/tests/test_data_loading.py

```diff
@@ -42,13 +44,15 @@ def test_load_image(mosviz_helper, mos_image):
     assert PRIHDR_KEY not in dc_1.meta
     assert dc_1.meta['RADESYS'] == 'ICRS'
 
-    table = mosviz_helper.app.get_viewer('table-viewer')
+    table = mosviz_helper.app.get_viewer(mosviz_helper._default_table_viewer_reference_name)
     table.widget_table.vue_on_row_clicked(0)
 
-    data = mosviz_helper.app.get_data_from_viewer('image-viewer')
+    data = mosviz_helper.app.get_viewer(mosviz_helper._default_image_viewer_reference_name
+                                        ).data()
 
-    dataval = data[f"{label} 0"]
-    assert isinstance(dataval, CCDData)
+    assert len(data) == 1
+    dataval = data[0]
+    assert isinstance(dataval, Data)
```

Comment:
Wait... is this PR supposed to modify the data type that gets returned? This looks like a behavior change.

Reply:
The intention I read from this test is to check if the data was loaded into the viewer properly, less so the actual class. That why I couldn't swap this out with `get_data` because there's no way to filter on which viewer the data are in.

For that reason, I had to do something a little different (call `viewer.data()` instead). I don't think it was guaranteed this would result in the same class, but it verifies the actual purpose of this test

Reply:
I found a way to keep the old behavior. Will push a commit.